source/accessibility.html.md.erb

```diff
@@ -7,7 +7,7 @@ hide_in_navigation: true
 
 # Accessibility
 
-This accessibility statement applies to the GDS Way at [https://gds-way.digital.cabinet-office.gov.uk/](https://gds-way.digital.cabinet-office.gov.uk/).
+This accessibility statement applies to the GDS Way at <https://gds-way.digital.cabinet-office.gov.uk/>.
```

Comment:
Ooh, nice, I didn't know this was a thing!

Reply:
Huh, TIL ([example from markdown guide](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses)), cool